themes/default/content/registry/packages/azure-native-v2/from-v1-to-v2/_index.md

```diff
@@ -0,0 +1,218 @@
+
+A full list of default version changes can be found in the [top-level resource versions](./top-level-resource-versions).
+
+To continue using the previous Azure API version of a resource:
+
+1. Check the documentation in your IDE or our [registry API docs](https://www.pulumi.com/registry/packages/azure-native-v2/) which identifies the previous version for each resource. For example: `Azure REST API Version: 2022-05-01. Prior API version in Azure Native 1.x: 2020-07-17-preview`
+2. Import the previous version of the resource. These are available in the version-specific sub-folders of the SDK.
+
+Below are examples of changing an import to use an explicit version in each language.
+
+{{< chooser language "typescript,python,csharp,go,yaml" >}}
+
+{{% choosable language typescript %}}
+
+```typescript
+- import { EventSubscription } from "@pulumi/azure-native/eventgrid";
++ import { EventSubscription } from "@pulumi/azure-native/eventgrid/v20200601";
+```
+
+{{% /choosable %}}
+{{% choosable language python %}}
+
+```python
+- from pulumi_azure_native import eventgrid
++ from pulumi_azure_native.eventgrid import v20200601 as eventgrid20200601
+```
+
+{{% /choosable %}}
+{{% choosable language csharp %}}
+
+```csharp
+- using EventGrid = Pulumi.AzureNative.EventGrid;
++ using EventGrid20200601 = Pulumi.AzureNative.EventGrid.V20200601;
+```
+
+{{% /choosable %}}
+{{% choosable language go %}}
+
+```go
+- import eventgrid "github.com/pulumi/pulumi-azure-native-sdk/eventgrid/v2"
++ import eventgrid20200601 "github.com/pulumi/pulumi-azure-native-sdk/eventgrid/v2/v20200601”
+```
+
+{{% /choosable %}}
+{{% choosable language yaml %}}
+
+```yaml
+- type: azure-native:eventgrid:Topic
++ type: azure-native:eventgrid/v20200601:Topic
+```
+
+{{% /choosable %}}
+{{< /chooser >}}
+
+#### User Assigned Identity Inputs
+
+`User assigned identity` inputs are now represented as a simple string array in each language instead of a map type. Where you are referencing user assigned identity inputs, you will need to update the syntax to resolve the error.
+
+```typescript
+import * as resources from "@pulumi/azure-native/resources";
+import * as managedidentity from "@pulumi/azure-native/managedidentity";
+import * as storage from "@pulumi/azure-native/storage";
+
+// Create a resource group
+const resourceGroup = new resources.ResourceGroup("my-resource-group");
+
+// Create a user-assigned managed identity
+const userAssignedIdentity = new managedidentity.UserAssignedIdentity("my-user-assigned-identity", {
+    resourceGroupName: resourceGroup.name,
+});
+
+// Create a storage account that references the user-assigned managed identity
+const storageAccount = new storage.StorageAccount("mystorageaccount", {
+    resourceGroupName: resourceGroup.name,
+    kind: "StorageV2",
+    location: resourceGroup.location,
+    identity: {
+        type: "UserAssigned",
+        userAssignedIdentities: [userAssignedIdentity.id],
```

Comment:
Should we add examples in other languages?

Reply:
I will add examples in other languages at a later date once [this](https://github.com/pulumi/pulumi/issues/13163) is resolved.